app/views/migration/_form.html.erb

```diff
@@ -54,14 +54,15 @@
             <%= wr.input(:dc_type) %>
             <%= wr.input(:dcterms_created, as: :date, html5: true, input_html: date_format_fallback_attributes) %>
             <%= wr.input(:dc_creator) %>
+            <%= wr.input(:edm_rights, as: :radio_buttons, collection: edm_rights_options, checked: wr.object&.edm_rights&.id, wrapper_html: { 'data-error-msg-key': 'contribute.campaigns.migration.form.validation.web-resource-license' }) %>
```

Comment:
Is it necessary to specify `checked` here?

Reply:
yes

Reply:
Change it to `wr.input(:edm_rights_id)`, update the permitted attribute in the controller, and `checked` can be removed, plus fewer MongoDB queries need be made on the edit action.